transfers/transfer.py

```diff
@@ -11,12 +11,15 @@
 import base64
 import logging
 import logging.config  # Has to be imported separately
+import models
```

Comment:
In python3 the import will fail, because imports are absolute by default and there's nothing called 'models' in the global packages.  Does adding `absolute_import` to the `from __future__ import` line fix it?